### PR TITLE
Update assignedaccess-csp.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/1954
Added note about assigned access.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/client-management/mdm/assignedaccess-csp.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ For a step-by-step guide for setting up devices to run in kiosk mode, see [Set u
 > [!Warning]
 > You can only assign one single app kiosk profile to an individual user account on a device. The single app profile does not support domain groups.
 
+> [!Note]
+> If the application runs in assigned access mode, when the app calls KeyCredentialManager.IsSupportedAsync and it returns false on the first run, try invoking the settings screen to have the user select a convenience PIN to use with Windows Hello. This is the settings screen that is hidden by the application running in assigned access mode. This means you can only use Windows Hello if you first leave Assigned Access. The user must then select his/her convenience pin and then go into Assigned Access again.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/assignedaccess-csp.md]:
```suggestion
> If the application calls KeyCredentialManager.IsSupportedAsync when it is running in assigned access mode and it returns false on the first run, invoke the settings screen and select a convenience PIN to use with Windows Hello. This is the settings screen that is hidden by the application running in assigned access mode. You can only use Windows Hello if you first leave assigned access mode, select your convenience pin, and then go back into assigned access mode again. 
```

**Comment by e0i** [comment]
@MariciaAlforque Kindly review this PR based on issue #1954 Thanks.

**Comment by e0i** [comment]
@MariciaAlforque Would you be able to review and merge this PR? We would like to follow-up. Thanks.

**Comment by e0i** [comment]
@Justinha Would you assist in having this PR reviewed and resolved? Thanks.

**Pull request merged**